README.md

```diff
@@ -86,57 +88,68 @@ Given this command, it will start a local development server and display the add
 
 You may supply a configuration file, in YAML format, to control various options.
 
-You can modify (or create) the `.redocly.yaml` file in the directory from which you are going to run the validator. Also, you can provide the path to the configuration file name other than `.redocly.yaml` by using `--config` option when running the @redocly/openapi-cli.
+You can modify (or create) the `.redocly.yaml` file in the directory from which you are going to run the linter. Also, you can provide the path to the configuration file name other than `.redocly.yaml` by using `--config` option when running the @redocly/openapi-cli.
 
-From a high-level, there are two configurable features: codeframes and rules.
-```yaml
-lint:
-  codeframes: on
-  rules:
-  ...
-```
+From a high-level, there are two configurable features: plugins and rules.
 
-### Codeframes
-
-Codeframes are enabled by default.  You may disable them by setting the value to `off`.
-
-```yaml
-lint:
-  codeframes: off
-  rules:
-  ...
-```
+See more about [plugins](docs/plugins.md)
 
 ### Rules
 
 Rules control validations used on the API definition.  You may customize them (and even extend them), or you may utilize the default configuration.
 
-Below is the default config:
+#### Extends
+
+By default all rules from the `recommended` ruleset are enbaled if nothing else is specified.
+You can extend other ruleset or disable, enalbe other rules:
+
 
 ```yaml
 lint:
-  codeframes: on
+  extends:
+    - recommended
   rules:
-    oas3-schema: on
-    path-param-exists: on
-    operation-2xx-response: on
-    unique-parameter-names: on
-    no-unused-schemas: on
-    operation-operationId-unique: on
-    path-declarations-must-exist: on
-
-    camel-case-names: off
-    api-servers: off
-    license-url: off
-    no-extra-fields: off
-    operation-description: off
-    operation-operationId: off
-    operation-tags: off
-    provide-contact: off
-    servers-no-trailing-slash: off
+    operation-2xx-response: off
+
+Below is the `recommended` preset:
 
-    bundler: off
-    debug-info: off
+```yaml
+  rules:
+    info-description: warn
+    info-contact: off
+    info-license: off
+    info-license-url: off
+
+    tag-description: warn
+    tags-alphabetical: off
+
+    no-server-example.com: warn
+    no-server-trailing-slash: error
+    no-empty-servers: warn
+
+    parameter-description: off
+    no-path-trailing-slash: error
+    path-declaration-must-exist: error
+    path-not-include-query: error
+    path-parameters-defined: error
+    operation-description: off
+    operation-2xx-response: warn
+    operation-operationId-unique: error
```

Comment:
What about `operation-operationId`? I see it is not in the recommended currently. I commented below that I think it should be.